### PR TITLE
Add digital asset link file

### PR DESCRIPTION
**Pull request by kyleshay**

### CODE REVIEW

File: src/android-config.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "relation": ["delegate_permission/common.handle_all_urls"],
+    "target": {
+      "namespace": "android_app",
+      "package_name": "com.destinyitemmanager.app",
```
Comment (bhollis) [inline src/android-config.json]: Will this work for both app and beta?

Comment (kyleshay) [reply src/android-config.json]: They'd need different configs. Was thinking we would just  have it for stable. If people want beta they can still use it as they do now?

Comment (bhollis) [reply src/android-config.json]: Sure. In that case can we only copy this file if the env is 'release'?

File: config/webpack.js [hunk 2]
```diff
@@ -401,6 +401,10 @@ module.exports = (env) => {
         'process.env': JSON.stringify({ NODE_ENV: 'production' })
       }),
 
+      new CopyWebpackPlugin([
```
Comment (bhollis) [inline config/webpack.js]: This will still copy for beta (env != 'dev'). You need a special check just for `env.release`.

**Pull request merged**